core/src/dpdk/port.rs

```diff
@@ -149,9 +160,29 @@ impl PortQueue {
                     // the ones already sent first and try again on the rest.
                     let drained = packets.drain(..sent as usize).collect::<Vec<_>>();
 
+                    //
+                    #[cfg(feature = "pcap-dump")]
+                    {
+                        if let Ok(pcap) = Pcap::append(
+                            format!("{:?}-{:?}-tx.pcap", self.port_id, CoreId::current()).as_str(),
+                        ) {
+                            pcap.write_packets(&drained);
+                        }
```

Comment:
if we fail to open or fail to write the packet here, we can generate an error log here to indicate the failure to pcap dump. it will be pretty noisy if things go wrong but I think we want that.

Reply:
ok.